### ISSUE TITLE
 On arrowing up / down JAWS18 / IE 11 does not read aria-label on element

### ISSUE THREAD

**Issue by devarshipant** [FreedomScientific/VFO-standards-support#73]
### Summary

> JAWS18 / IE 11 only reads aria-label on tabbing to heading, not on arrowing up / down.

> Example:
> 1. Go to https://codepen.io/devpant/pen/VxjEvZ

### Expected result

> JAWS 18 should read aria-label on arrowing down / up to heading.

### Actual result

> aria-label on heading element is only read on tabbing to it.

### Example

> https://codepen.io/devpant/pen/VxjEvZ

**Comment by patrickhlauke** [comment]
https://developer.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/

**Comment by stevefaulkner** [comment]
@devarshipant use of `tabindex=0` on elements with non interactive roles is generally problematic for AT, for example when tabbed to using NVDA/IE11 the heading is announced as 'edit'. As this markup pattern is not recommended and can cause odd interactions, it is discouraged. What is your use case?

**Comment by devarshipant** [comment]
Updated issue title to correctly reflect the use case. Please feel free to revise / close the issue if needed.

**Comment by lamador** [comment]
When using JAWS version 2020.2004.66 with IE11 version 11.778.18362.0, JAWS reads the visual text of the heading elements and ignores the "aria-label".  This behavior is described in the "Practical Support: aria-label, aria-labelledby and aria-describedby" in the W3C [Using ARIA](https://www.w3.org/TR/using-aria/#practical-support-aria-label-aria-labelledby-and-aria-describedby) document.
